feder/monitorings/templates/monitorings/monitoring_draft_list.html

```diff
@@ -0,0 +1,38 @@
+{% extends 'monitorings/base_monitoring_detail.html' %}
+{% load i18n humanize %}
+{% block content_object %}
+    {% include 'monitorings/_tabs.html' with tab='drafts' %}
+    <h3 class="sr-only">{% trans 'Drafts' %}</h3>
+    {% if object_list %}
+        {% for object in object_list %}
+            {% include 'monitorings/_letter.html' %}
```

Comment:
Ja bym napisał wprost ``{% include 'monitorings/_letter.html' with object=object %}``. Nie jest to konieczne, ale oddaje to istotę skutecznie.

Reply:
poprawione